pkgs/desktops/xfce4-13/xfce4-terminal/default.nix

```diff
@@ -0,0 +1,15 @@
+{ mkXfceDerivation, gtk3, libxfce4ui, vte }:
+
+mkXfceDerivation rec {
+  category = "apps";
+  pname = "xfce4-terminal";
+  version = "0.8.6";
+
+  sha256 = "1a0b2ih552zhbbx1fc5ad80nafvkc5my3gw89as4mvycnhyd5inj";
+
+  buildInputs = [ gtk3 libxfce4ui vte ];
```

Comment:
I noticed in #33154 that `wrapGAppsHook` is required.

Reply:
It isn't, at least in my branch. Just built `xfce4-13.xfce4-terminal`, runs without `wrapGAppsHook` set.

![screenshot_2017-12-29_00-18-06](https://user-images.githubusercontent.com/1643293/34426382-cae1604a-ec2d-11e7-8ad4-9a3362246e69.png)

Reply:
Usually program runs without it, but lack some features being unable to find the files needed. For example ```xfce4-screenshooter``` runs and works but does not upload screenshots to imgur.com

Reply:
I see, then I'll wrap all Xfce apps.